### PR TITLE
Shell examples for episode 091 & 092

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: sample_code/ep091/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+### Bash
+
+Session 1:
+
+```bash
+rm -rf /
+
+podman run --rm -ti ubuntu:focal bash
+# dokcer run --rm -ti ubuntu:focal bash
```
Comment (asottile) [inline sample_code/ep091/README.md:15]: docker

**Review by asottile — APPROVED** [review]

File: sample_code/ep091/README.md (hunk 1 above)
Comment (int3l) [reply sample_code/ep091/README.md]: Sorry for the typo, feeling a bit ill.

Comment (asottile) [reply sample_code/ep091/README.md]: no problem <3

**Pull request merged**